src/grandorgue/GOOrganController.cpp

```diff
@@ -686,6 +690,33 @@ bool GOOrganController::IsToImportCombinationsFor(
   return isToImport;
 }
 
+static std::vector<char> loadFileBytes(const wxString &file) {
+  wxFileInputStream fileInputStream(file);
+  if (!fileInputStream.IsOk()) {
+    throw wxString::Format(_("Failed to open '%s'"), file);
+  }
+  std::vector<char> content(fileInputStream.GetSize());
+  if (!fileInputStream.ReadAll(&content[0], content.size())) {
+    throw wxString::Format(_("Failed to read '%s'"), file);
+  }
+  return content;
+}
+
+static wxString loadFileTextWithEncodingDetection(const wxString &file) {
+  std::vector<char> content = loadFileBytes(file);
+  std::array<char, 3> utf8bom = {'\xEF', '\xBB', '\xBF'};
+  if (
+    content.size() >= utf8bom.size()
+    && std::equal(utf8bom.begin(), utf8bom.end(), content.begin())) {
```

Comment:
If the file contains soma another BOM (not utf8), this method tries to use *wxConvCurrent instead of wxConvAuto()